Tests/RevenueCatUITests/PaywallsV2/VariableHandlerV2Tests.swift

```diff
@@ -64,7 +64,58 @@ class VariableHandlerV2Test: TestCase {
             "num_year_two": "%d years",
             "num_year_few": "%d years",
             "num_year_many": "%d years",
-            "num_year_other": "%d years"
+            "num_year_other": "%d years",
+            "num_days_short": "%1$s%2$s",
+            "num_weeks_short": "%1$s%2$s",
+            "num_months_short": "%1$s%2$s",
+            "num_years_short": "%1$s%2$s"
```

Comment:
These are the new values coming from the backend

Even though the first is will be rendered as a integer (like "3 months"), we have to make it a string here because of a Swift bug that can cause a `EXC_BAD_ACCESS` if there are ordered types

I will link to the feedback when its created